src/index.js

```diff
@@ -6,29 +6,28 @@ import { ApolloClient } from 'apollo-client'
 import { HttpLink } from 'apollo-link-http'
 import { InMemoryCache } from 'apollo-cache-inmemory'
 
-import App from 'screens/App/screens/App';
-import registerServiceWorker from './registerServiceWorker';
 import Apollo from 'shared/utilities/apollo'
-import './index.css';
+import AppEntry from './screens/App/components'
+import registerServiceWorker from './registerServiceWorker'
 
-const httpLink = new HttpLink({ 
+const httpLink = new HttpLink({
   uri: 'https://data.blabbermouth96.hasura-app.io/v1',
   // credentials: 'include',
   fetchOptions: {
     mode: "no-cors",
-  }
+  },
 })
 
 
 const client = new ApolloClient({
   link: Apollo.AuthLink.concat(httpLink),
-  cache: new InMemoryCache()
+  cache: new InMemoryCache(),
 })
 
 ReactDOM.render(
   <ApolloProvider client={client}>
-    <App />
-  </ApolloProvider>, 
+    <AppEntry />
```

Comment:
Can we render App the way it was? The App component housed the routes that should be secured and the App had other routes that should be left unauthenticated.